is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
+      if (elementToCompare === currentElement) {
+        isAllUnique = false;
+      }
+    });
+  });
+
+  return isAllUnique;
+};
+
+export default isAllUniqueElements;
```

Comment:
Is this line necessary? From the code I see here, I can't tell why it is needed.